articles/includes/code/dotnet-formflow-advanced.cs

```diff
@@ -1,17 +1,24 @@
 // <validationFunction>
-.Field(nameof(Toppings),
-    validate: async (state, value) =>
-    {
-        var values = ((List<object>)value).OfType<ToppingOptions>();
-        var result = new ValidateResult { IsValid = true, Value = values };
-        if (values != null && values.Contains(ToppingOptions.Everything))
-        {
-            result.Value = (from ToppingOptions topping in Enum.GetValues(typeof(ToppingOptions))
-                            where topping != ToppingOptions.Everything && !values.Contains(topping)
-                            select topping).ToList();
-        }
-        return result;
-    })
+public static IForm<SandwichOrder> BuildForm()
+{
+    ...
+    return new FormBuilder<SandwichOrder>()
+        .Message("Welcome to the sandwich order bot!")
+        ...
```

Comment:
Please remove the "..." here and instead add these four lines that precede the Toppings line in the sample app:

.Field(nameof(Sandwich))
.Field(nameof(Length))
.Field(nameof(Bread))
.Field(nameof(Cheese))